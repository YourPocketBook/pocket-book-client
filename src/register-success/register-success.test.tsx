import { render } from "@testing-library/react";
import React from "react";
import ReactDOM from "react-dom";
import { RegisterSuccess } from ".";

describe("Home Page", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<RegisterSuccess />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders correctly", () => {
    const { container } = render(<RegisterSuccess />);

    expect(container).toMatchSnapshot();
  });
});
