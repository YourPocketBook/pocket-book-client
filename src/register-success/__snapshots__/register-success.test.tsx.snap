// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<div>
  <div
    class="background"
  >
    <nav
      class="navbar navbar"
    >
      <a
        class="navbar-brand"
      >
        PocketBook
      </a>
    </nav>
    <div
      class="centreContainer container"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-6 offset-lg-3"
        >
          <div
            class="centreBox"
          >
            <div
              class="row"
            >
              <div
                class="col"
              >
                <h1
                  class="text-center"
                >
                  Register Success
                </h1>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col"
              >
                <p>
                  Thanks, you have registered successfully. You now need to check your inbox, where you will shortly receive a confirmation email which will explain what to do next.
                </p>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-sm-4 offset-sm-4"
              >
                <a
                  class="btn btn-dark-green"
                  href="/"
                  id="home-button"
                >
                  Home
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <nav
      class="fixed-bottom footer navbar"
    >
      <span
        class="navbar-text"
      >
        Design Copyright Â© 
        2020
         Tony Richards.
      </span>
      <ul
        class="nav"
      >
        <li
          class="nav-item"
        >
          <a
            class="nav-link text-dark"
            href="/privacy"
          >
            Privacy Policy
          </a>
        </li>
        <li
          class="nav-item"
        >
          <a
            class="nav-link text-dark"
            href="/tsandcs"
          >
            Terms of Use
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
